doc/quickstart.rst

```diff
@@ -18,10 +18,249 @@ examples below:
+	lat                            0.0
+	lon                            0.0
+	org                            arch-example-data
+	project                        05612d90-afc5-47be-b6a6-304b26c56488
+	sg                             single-temp-sensor-v1-1-0
+	slug                           d--0000-0000-0000-00fd
+	state                          N1
+	template                       pod-1gg1-es1-v2-0-0
+
+	Properties
+	----------
+
+	Stream Summaries
+	----------------
+	System Data 5C00                         s--0000-00c8--0000-0000-0000-00fd--5c00
+	System Data 5C01                         s--0000-00c8--0000-0000-0000-00fd--5c01
+	System Data 5C02                         s--0000-00c8--0000-0000-0000-00fd--5c02
+	System Data 5C03                         s--0000-00c8--0000-0000-0000-00fd--5c03
+	Temperature                              s--0000-00cb--0000-0000-0000-00fd--500f
+	System Data 5800                         s--0000-00cb--0000-0000-0000-00fd--5800
+	System Data 5A05                         s--0000-00cb--0000-0000-0000-00fd--5a05
+	System Data 5C00                         s--0000-00cb--0000-0000-0000-00fd--5c00
+	System Data 5C01                         s--0000-00cb--0000-0000-0000-00fd--5c01
+	System Data 5C02                         s--0000-00cb--0000-0000-0000-00fd--5c02
+	System Data 5C04                         s--0000-00cb--0000-0000-0000-00fd--5c04
+	System Data 5C05                         s--0000-00cb--0000-0000-0000-00fd--5c05
+
+	Stream Counts
+	-------------
+	s--0000-00c8--0000-0000-0000-00fd--5c00                   2 points      0 events
+	s--0000-00c8--0000-0000-0000-00fd--5c01                   3 points      0 events
+	s--0000-00c8--0000-0000-0000-00fd--5c02                   2 points      0 events
+	s--0000-00c8--0000-0000-0000-00fd--5c03                   1 points      0 events
+	s--0000-00cb--0000-0000-0000-00fd--500f               27322 points      0 events
+	s--0000-00cb--0000-0000-0000-00fd--5800                4585 points      0 events
+	s--0000-00cb--0000-0000-0000-00fd--5a05                8516 points      0 events
+	s--0000-00cb--0000-0000-0000-00fd--5c00                  41 points      0 events
+	s--0000-00cb--0000-0000-0000-00fd--5c01                4425 points      0 events
+	s--0000-00cb--0000-0000-0000-00fd--5c02                4422 points      0 events
+	s--0000-00cb--0000-0000-0000-00fd--5c04                4585 points      0 events
+	s--0000-00cb--0000-0000-0000-00fd--5c05                4585 points      0 events
+
+
+This report tells us that we have a lot of system data available about this device
+and just one user friendly temperature stream named `Temperature`.  Let's view the
+temperature data.
+
+.. code:: bash
+	
+	$ analytics-host -t stream_overview ./freezer.hdf5 -a stream=temp -c -o temperature -a units=Celsius
+	Rendered report to: temperature.html
+
+You should have a file in your local directory named `temperature.html`, which when you open in a browser
```

Comment:
I think this command is dropped on the reader without a little introduction:

"The `-o` option tells the tool to output a static HTML file to interact with the report." 

Can you also move the two -a arguments at the end for consistency?